packages/l/linphone-desktop/package.yml

```diff
@@ -0,0 +1,56 @@
+name       : linphone-desktop
+version    : 5.2.4
+release    : 1
+source     :
+    - git|https://gitlab.linphone.org/BC/public/linphone-desktop.git : d69e033508265f756fb47b39b76b7d589be159c0
```

Comment:
The project uses Git tags, so we should use them if at all possible: `https://gitlab.linphone.org/BC/public/linphone-desktop/-/archive/5.2.4/linphone-desktop-5.2.4.tar.bz2`